### PR TITLE
Initial scaffolding for the python package

### PR DESCRIPTION
**Pull request by JsseL**
Scaffolding ported from https://github.com/andersinno/cmsplugin-andersblog.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Image carousel plugin for django-cms
```
Comment (akx) [inline LICENSE]: Replace with the MIT license boilerplate, please.

File: MANIFEST.in [hunk 2]
```diff
@@ -0,0 +1,4 @@
+include README.md
+recursive-include carousel *.*
```
Comment (akx) [inline MANIFEST.in]: These should be `cmsplugin_carousel_ai` (directory name).

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+import subprocess
+from distutils.core import setup
+
+git_version = 'UNKNOWN'
```
Comment (akx) [inline setup.py]: No need for the git version malarkey imho, just `version='0.0.0'` in the `setup()` invocation.

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+try:
+    git_version = subprocess.check_output(['git', 'rev-parse', '--verify', '--short', 'HEAD']).decode('ascii').strip()
+except subprocess.CalledProcessError:
+    pass
+
+setup(
+    name='cmsplugin_carousel_ai',
```
Comment (akx) [inline setup.py]: This should be `cmsplugin-carousel-ai` (dashes, not underscores).

Comment (JsseL) [reply setup.py]: Where does this have an effect?

Comment (akx) [reply setup.py]: That's the name of the package on PyPI.

File: LICENSE (hunk 1 above)
Comment (JsseL) [reply LICENSE]: Replaced..

File: LICENSE [hunk 5]
```diff
@@ -0,0 +1,8 @@
+The MIT License (MIT)
+Copyright (C) 2016 Anders Innovationsn Oy
```
Comment (akx) [inline LICENSE]: Typo in company name...

Comment (JsseL) [reply LICENSE]: Fixed. More comments?

**Comment by akx** [comment]
👌

**Pull request merged**